README.md

```diff
@@ -158,146 +255,96 @@ Wenn der Request nicht erfolgreich verarbeitet werden konnte, liefert die Schnit
-* auszahlungsbetrag - BigDecimal
-  * die erlaubten Werte sind eingeschränkt - siehe dazu die Query *grenzen*  
-* laufzeitInMonaten - Int
-  * die erlaubten Werte sind eingeschränkt - siehe dazu die Query *grenzen*  
-* finanzierungszweck - String
-  * hierbei handelt es sich um eine Aufzählung (ENUM) mit folgenden Ausprägungen
-    * "UMSCHULDUNG"
-    * "FREIE_VERWENDUNG"
-    * "FAHRZEUGKAUF"
-    * "MODERNISIEREN"  
-  * wenn nicht angegeben, wird das beste Angebot über alle Finanzierungszwecke hinweg ermittelt
-* datenkontext - String
-  * hierbei handelt es sich um eine Aufzählung (ENUM) mit folgenden Ausprägungen
-    * "ECHTGESCHAEFT"
-    * "TESTUMGEBUNG"
-  * wenn nicht angegeben, wird TESTUMGEBUNG angenommen
-
-### BestesAngebot: gewünschte Felder
+    grenzen(partnerId: <partnerId>){
+        <gewünschte Felder>
+    }
     
-    Whitespace-separierte Liste folgender Felder
-
-       | Feldname           | Typ                                   |
-       | ------------------ | ------------------------------------- |
-       | produktanbieter    | String                                |   
-       | produktbezeichnung | String                                |     
-       | gesamtbetrag       | BigDecimal: Betrag in Euro            |  
-       | nettokreditbetrag  | BigDecimal: Betrag in Euro            | 
-       | laufzeitInMonaten  | Int                                   | 
-       | sollzins           | BigDecimal: 100-basierter Prozentsatz |  
-       | effektivzins       | BigDecimal: 100-basierter Prozentsatz |  
-       | monatlicheRate     | BigDecimal: Betrag in Euro            |    
-       | letzteRate         | BigDecimal: Betrag in Euro            |    
-       | ------------------ | ------------------------------------- |
+### Parameter
+
+| Parametername| Typ | Bemerkung | 
+|--------------|-----|------------|
+| partnerId | String | Die PartnerId ist 5-stellig und identifiziert eine Plakette aus dem Europace-Partnermanagement. <br/>Die angegebene PartnerId muss unterhalb der PartnerId des JWTs liegen oder mit ihr identisch sein. |
+| auszahlungsbetrag | BigDecimal|  Die erlaubten Werte müssen innerhalb der Grenzen der partnerId sein.|
+| laufzeitInMonaten | Integer |  Die erlaubten Werte müssen innerhalb der Grenzen der partnerId sein.|
+| finanzierungszweck | "UMSCHULDUNG" <br/> "FREIE_VERWENDUNG" <br/> "FAHRZEUGKAUF" <br/> "MODERNISIEREN" |  wenn nicht angegeben, wird das beste Angebot über alle Finanzierungszwecke hinweg ermittelt
+| datenkontext | "ECHTGESCHAEFT" <br/> "TESTUMGEBUNG" |wenn nicht angegeben, wird TESTUMGEBUNG angenommen|
+  
+### Gewünschte Felder
+
+Für eine bessere Lesbarkeit wird das Gesamtformat in *Typen* aufgebrochen, die an anderer Stelle definiert sind, aber an verwendeter Stelle eingesetzt werden müssen.  
+Es gibt die Scalare `Euro` und `Prozent`, die jeweils Wrapper für BigDecimal sind.
+
     
-### BestesAngebot: Response Format
+#### Angebot
```

Comment:
Wir brauchen noch einen Abschnitt für die Pflichtfelder bei der Anfrage, muss nicht umbedingt in diesem PR sein, sollten wir aber nicht vergessen.

Reply:
Guter Punkt, ich habe ein Checklistenpunkt auf unserem Ticket hinzugefügt